ssl/ssl_test.cc

```diff
@@ -2584,15 +2582,23 @@ GU5F4zAKBggqhkjOPQQDAgNJADBGAiEAiiNowddQeHZaZFIygwe6RW5/WG4sUXWC
 dkyl9CQzRaYCIQCFS1EvwZbZtMny27fYm1eeYciY0TkJTEi34H1KwyzzIA==
 -----END CERTIFICATE-----
 )");
-  ASSERT_TRUE(root);
+  EXPECT_TRUE(root);
+  return root;
+}
+
+static bssl::UniquePtr<EVP_PKEY> GetLeafKey() {
   bssl::UniquePtr<EVP_PKEY> leaf_key = KeyFromPEM(R"(
 -----BEGIN PRIVATE KEY-----
 MIGHAgEAMBMGByqGSM49AgEGCCqGSM49AwEHBG0wawIBAQQgj5WKHwHnziiyPauf
 7QukxTwtTyGZkk8qNdms4puJfxqhRANCAARNrkhxabALDlJrHtvkuDwvCWUF/oVC
 hr6PDITHi1lDlJzvVT4aXBH87sH2n2UV5zpx13NHkq1bIC8eRT8eOIe0
 -----END PRIVATE KEY-----
 )");
-  ASSERT_TRUE(leaf_key);
+  EXPECT_TRUE(leaf_key);
```

Comment:
minor nit: should we ASSERT here? the test can't meaningfully progress without the leaf key, right?

Reply:
Initially tried that, but ASSERT isn't allowed in abstracted test functions unless the function signature is `void` :(

Reply:
ah, interesting. that's actually a really solid decision by the gtest authors.